sender.go

```diff
@@ -102,6 +104,12 @@ func (sender *Sender) Close() {
 func (sender *Sender) start() {
 	sender.conn.PayloadType = websocket.BinaryFrame
 	go (func() {
+		defer func() {
```

Comment:
The other panic handler newly introduced in this PR calls `fatalError()` in the panic handler.  Should this call it here as well?  I'm just wondering if was it left out on purpose or if was it an oversight?